sstables/trie/trie_writer.hh

```diff
@@ -35,6 +35,7 @@
 #include <seastar/util/log.hh>
 #include <map>
```

Comment:
> Note: this might be overengineered. Maybe we should force the
user to linearize the key. Maybe caring so much about fragmented
keys as first-class citizens is the wrong thing to do.

1. keys are not fragmented as long as they're outside lsa (even stored as bytes, not managed_bytes)
2. I'd like to eventually convert to comparably_bytes early (at the coordinator)

But we can undo these later.

Reply:
Yes, undoing the fragmentation is always easier than introducing it.